### DISCUSSION TITLE
Removed an entry sensor and it still shows as an accessory and as "open"

### DISCUSSION THREAD

**Discussion by jwonder** [homebridge-simplisafe3/homebridge-simplisafe3#436] [Q&A] [answered]
Hello, I removed an entry sensor and replaced it with another and yet it still shows in the accessory list on HomeBridge.  I tried to add the sensors serial number to the excluded list and that did not work.  I surmise this is still stored in a database somewhere and I need to delete the row, but I don't know where it might be.

Any help is appreciated as it is throwing off my automations because its coming back as an "open sensor".

Thanks!!!

**Comment by shamoon** [comment, marked as answer]
Try unchecking the "Persist Accessories" button and then restart and then you can "Remove Single Cached Accessory" in homebridge if it still persists. You'll probably want to re-check the Persist option again after you're done

**Reply by jwonder** [reply]
That worked 100%!  Thank you, I am still learning my way around HomeBridge and I love your plugin!!!  Thanks!!!
